api/controllers/game/move.js

```diff
@@ -33,14 +35,20 @@ module.exports = async function (req, res) {
 
     return res.ok();
   } catch (err) {
-    // unlock game if failing due to validation
+    // ensure the game is unlocked
     if (game?.lock) {
       try {
         await sails.helpers.unlockGame(game.lock);
       } catch (err) {
         // fall through for generic error handling
       }
     }
-    return res.badRequest({ message: err.message });
+
+    switch (err?.code) {
+      case 'FORBIDDEN':
+        return res.forbidden({ message: err.message });
+      default:
+        return res.badRequest({ message: err.message });
+    }
```

Comment:
Created this flow to allow endpoints that lock the game to share logic for unlocking the game when an error occurs and then to still have their own individual handlers. I think in order to more fully flesh this out, we should replace all the raw errors with a new subclass called `BadRequestError` which would set the code to `'BAD_REQUEST'`. Then we could send 400 on the bad request errors and default to 500 instead as our catch-all, which I think would better capture the space of things that can go wrong